Source/Core/UICommon/GameFileCache.h

```diff
@@ -30,7 +30,7 @@ class GameFileCache
   void ForEach(std::function<void(const std::shared_ptr<const GameFile>&)> f) const;
 
   size_t GetSize() const;
-  void Clear();
+  void Clear(bool delete_on_disk);
```

Comment:
This seems like something that should be an enum class rather than a bool, since `Clear(true);` is kind of vague in how it communicates what it's doing from a call site. LGTM otherwise though.